src/ts/Conviva.d.ts

```diff
@@ -39,28 +39,28 @@ declare namespace Conviva {
 
   class Client {
     constructor(settings: ClientSettings, systemFactory: SystemFactory);
-    adEnd(sessionKey: number): void;
-    adStart(sessionKey: number, adStream: Client.AdStream, adPlayer: Client.AdPlayer,
+    public adEnd(sessionKey: number): void;
```

Comment:
The changelog says that the type declarations were improved, but the only difference I can see here are unnecessary `public` modifiers, so semantically there is no difference and the changelog entry should be removed - or is there anything I have missed?

Reply:
The typings of `ConvivaAnalytics.ts` have been improved, also because of player v8. I did not touch the ambient typings of the Conviva API. E.g., `Player` was typed as `any` before.